ndmg/stats/qa_reg.py

```diff
@@ -56,61 +69,51 @@ def reg_mri_pngs(
     else:  # dim=3
         mr_data = mri_data
 
-    cmap1 = LinearSegmentedColormap.from_list("mycmap1", ["black", "magenta"])
-    cmap2 = LinearSegmentedColormap.from_list("mycmap2", ["black", "green"])
+    cmap1 = LinearSegmentedColormap.from_list("mycmap1", ["white", "magenta"])
+    cmap2 = LinearSegmentedColormap.from_list("mycmap2", ["white", "green"])
 
     fig = plot_overlays(atlas_data, mr_data, [cmap1, cmap2], minthr, maxthr, edge)
-
     # name and save the file
-    fname = os.path.split(mri)[1].split(".")[0] + ".png"
-    fig.savefig(outdir + "/" + fname, format="png")
+    fig.savefig(outdir + "/" + os.path.split(mri)[1].split(".")[0] + ".png", format="png")
```

Comment:
You don't include information about what `atlas` file is being used as a reference for the registration. Your output images are only titled after the `mri` (pink), without telling what the atlas image (green) is. This could cause confusion when looking at the output pngs

Reply:
see https://github.com/neurodata/ndmg/pull/372/commits/6ea410c29306aa2c148825fa6fadf0d6d85f7d13

Reply:
Use [gen_utils.get_filename](https://github.com/neurodata/ndmg/blob/staging-tmp/ndmg/utils/gen_utils.py#L643) instead of the code you wrote

Reply:
see https://github.com/neurodata/ndmg/pull/372/commits/989fdc844cee4c054b1f4a3c898acfe960818776